dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -680,6 +609,165 @@ def test_json_decode_after_update(self):
         assert deserialized.report().get("categorical", None) == False
 
 
+class TestColumnDataLabelerCompiler(unittest.TestCase):
+    def test_column_data_labeler_compiler_report(self):
+        structured_options = StructuredOptions()
+        structured_options.category.is_enabled = False
+        data1 = pd.Series(["2.6", "-1.8", "-2.3"])
+        compiler1 = col_pro_compilers.ColumnDataLabelerCompiler(
+            data1, structured_options
+        )
```

Comment:
we should use mocks on the class maybe, i assume we don't need the specifically for these tests to run a labeler